Wikipedia/Code/Article as a Living Document/ArticleAsLivingDocController.swift

```diff
@@ -68,27 +68,7 @@ class ArticleAsLivingDocController: NSObject {
     
     var articleAsLivingDocViewController: ArticleAsLivingDocViewController?
     
-    var shouldAttemptToShowArticleAsLivingDoc: Bool {
```

Comment:
Solving in this way should be fine. It looks like the old `EventLoggingService` was used as storage for which A/B testing bucket a user landed in for the article as a living document experiment.